samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -43,6 +43,26 @@ LOG_MODULE_REGISTER(app_lwm2m_client, CONFIG_APP_LOG_LEVEL);
 BUILD_ASSERT(sizeof(CONFIG_APP_LWM2M_SERVER) > 1,
 	     "CONFIG_APP_LWM2M_SERVER must be set in prj.conf");
 
+
+#if defined(CONFIG_LWM2M_CLIENT_UTILS_FIRMWARE_INSTANCE_MODEM)
+
+BUILD_ASSERT(
```

Comment:
Shouldn't those static asserts be a part of the library rather than the sample? They verify Kconfigs specified by the library.